src/snakemake/spawn_jobs.py

```
@@ -55,6 +56,11 @@ def _get_storage_provider_setting_items(self):
                     unparse = field.metadata.get("unparse", lambda value: value)
 
                     def fmt_value(tag, value):
+                        if callable(value):
+                            raise WorkflowError(
+                                f"Invalid setting for plugin {plugin_name}. Unable "
```

Ruff (0.8.2)

Function definition does not bind loop variable `plugin_name`

(B023)